docs/configuration.md

```diff
@@ -0,0 +1,71 @@
+# Configuring Spectral
+
+## Example config file (spectral.yaml)
+
+```yaml
+rules:
+    - rule-name:
```

Comment:
Why is this an array objects? I'm assuming the `rule-name` is unique, so we can do something like

```yml
rules:
  rule-name:
    option1: true
  rule-name2:
    option1: true
```

Reply:
Yeah, this readme is BS :) I have to update it. You're completely right about it, it's never even meant to be an array, it's a dictionary.